vendor/bat-native-ads/src/bat/ads/internal/filters/ad_history_confirmation_filter.cc

```diff
@@ -0,0 +1,91 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+#include <algorithm>
+#include <memory>
+#include <string>
+
+#include "bat/ads/internal/filters/ad_history_confirmation_filter.h"
+#include "bat/ads/confirmation_type.h"
+#include "bat/ads/ad_history_detail.h"
+
+#include "bat/ads/ads_history.h"
+
+namespace ads {
+
+bool IsConfirmationTypeOfInterest(const ConfirmationType& confirmationType) {

```

Comment:
`confirmationType` should use `camel_case`

Reply:
Done